packages/webapp/src/containers/Task/TaskAssignment/index.jsx

```diff
@@ -113,7 +113,7 @@ export default function TaskManagement({ history, match, location }) {
     Object.keys(data).forEach((key) => {
       if (assignTaskFields.includes(key)) {
         delete postData[key];
-        delete postData[already_completed];
+        delete postData['already_completed'];
```

Comment:
This looks good, thank you! I have two requests:
- Can we use the constant `ALREADY_COMPLETED`?
- It's not your code, but it's not necessary to do `postData['already_completed']` when it's already been deleted. It would be great if you could take it out from `forEach`.